engine/runtime-integration-tests/src/test/java/org/enso/compiler/dump/test/DocsGenerateTest.java

```diff
@@ -374,6 +374,35 @@ public void intersectionTypes() throws Exception {
         sig);
   }
 
+  @Test
+  public void typesWithError() throws Exception {
+    var code =
+        """
+        type A
+        type B
+        type C
+
+        one a:A -> A & B ! C = a
+        """;
+
+    var v = new MockVisitor();
+    generateDocumentation("Error", code, v);
+
+    assertEquals("One methods", 1, v.visitMethod.size());
+    assertEquals("No constructors", 0, v.visitConstructor.size());
+
+    var p = v.visitMethod.get(0);
+    assertNull("It is a module method", p.t());
+
+    var m = p.ir();
+    var sig = DocsVisit.toSignature(m);
+    assertEquals("one", m.methodName().name());
+    assertEquals(
+        "Generates vector with argument type as return type",
```

Comment:
This message is copy-pasted. Reword it to something like "Generates thrown dataflow errors in the signature"

Reply:
Merged without another CI cycle. Addressed in another PR: https://github.com/enso-org/enso/pull/12972/commits/b4de3cc82863d096f996cd60a9da73c502550914